packages/flutter_tools/test/general.shard/commands/format_test.dart

```diff
@@ -74,5 +74,30 @@ void main() {
       final String shouldNotFormatted = srcFile.readAsStringSync();
       expect(shouldNotFormatted, nonFormatted);
     });
+
+    testUsingContext('line-length', () async {
+      const int lineLength_short = 50;
+      const int lineLength_long = 120;
+      final String projectPath = await createProject(tempDir);
+
+      final File srcFile = fs.file(
+          fs.path.join(projectPath, 'lib', 'main.dart'));
+      final String nonFormatted = srcFile.readAsStringSync();
+      srcFile.writeAsStringSync(
+          nonFormatted.replaceFirst('main()',
+              'main(anArgument1, anArgument2, anArgument3, anArgument4, anArgument5)'));
+
+      final String nonFormattedWithLongLine = srcFile.readAsStringSync();
+      final FormatCommand command = FormatCommand();
+      final CommandRunner<void> runner = createTestCommandRunner(command);
+
+      await runner.run(<String>['format', '--line-length', '$lineLength_long', srcFile.path]);
+      final String shouldNotFormatted = srcFile.readAsStringSync();
```

Comment:
nit: `notFormatted` instead of `shouldNotFormatted`

Reply:
Fixed